src/components/Nav/Nav.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import homeIcon from '../../home.svg';
+import searchIcon from '../../search.svg';
+import './Nav.css';
+
+const Nav = () => {
+  return (
+    <nav>
+      <img src={homeIcon} />
+      <h1>Rancid Tomatillos</h1>
+      <img src={searchIcon} />
```

Comment:
We talked a little about this last night, but just writing it as a reminder to future selves that this will be a button at some point